drivers/sqlboiler-mysql/driver/mysql_test.go

```diff
@@ -18,13 +18,15 @@ import (
 	"os/exec"
 	"testing"
 
+	"github.com/stretchr/testify/require"
+
 	"github.com/volatiletech/sqlboiler/v4/drivers"
 )
 
 var (
 	flagOverwriteGolden = flag.Bool("overwrite-golden", false, "Overwrite the golden file with the current execution results")
 
-	envHostname = drivers.DefaultEnv("DRIVER_HOSTNAME", "localhost")
+	envHostname = drivers.DefaultEnv("DRIVER_HOSTNAME", "127.0.0.1")
```

Comment:
I assume this was changed by accident. Can we revert?

Reply:
Reverted.